Gemfile.lock

```diff
@@ -205,6 +206,8 @@ GEM
     nokogiri (1.11.2)
       mini_portile2 (~> 2.5.0)
       racc (~> 1.4)
+    nokogiri (1.11.2-x64-mingw32)
```

Comment:
Likewise with these other gem files changes, if they are unnecessary to get the tests working, suggest reverting it back.

Reply:
This was the same as the previous one.

Reply:
Can you do a git checkout to reset the Gemfile.lock and run bundle install again with the removal of that line above (in the gem file), I think then it will remove this change